submission.md

```diff
@@ -36,7 +36,7 @@ In order to submit a project to the Brainhack Global 2020, please follow these s
 1. Fill the necessary information as asked you to do so.
 1. Please make sure you give as clear and brief information about your project as possible sharing the necessary links that point reviewer/attendees to the right sources/repository and more detailed information about your project.
 1. Please make sure to follow the instructions in **<a href="https://github.com/brainhackorg/bhg-event-materials/blob/master/docs/leading_and_attending_to_brainhack_projects.md" target="_blank">How to Guideline for Leading and Attending to Brainhack Projects</a>** to increase the diversity and inclusivity in your project work and define the tasks, roles, and the skills that you are expecting from your attendees accordingly. Make sure you have room for the newcomers to give them a chance to learn from the seniors and share their own experiences.
-1. In the skill section you should list the labels that relate to the project. This is a very important step since this information will be use in our internal algorithm to match the participants with the projects based on the registration form. In case you don't put enough labels reviewers will add them at will.**<a href="https://github.com/brainhackorg/global2020/labels" target="_blank">labels list</a>**
+1. In the skill section you should list the labels that relate to the project. This is a very important step since this information will be use in our internal algorithm to match the participants with the projects based on the registration form. We recommend to add labels from the categories git_skills_label, modality_label, project_type_label, project_tools_skills_label, tools_label topic_label. In case you don't put enough labels reviewers will add them at will.**<a href="https://github.com/brainhackorg/global2020/labels" target="_blank">labels list</a>**
```

Comment:
```suggestion
1. In the skill section you should list the labels that relate to the project. This is a very important step since this information will be used in our internal algorithm to match the participants with the projects. We recommend to add labels from the following categories `git_skills_label, modality_label, project_type_label, project_tools_skills_label, tools_label, topic_label`. In case you don't put enough labels reviewers will add them at will.**<a href="https://github.com/brainhackorg/global2020/labels" target="_blank">labels list</a>**
```